learning_resources/etl/micromasters.py

```diff
@@ -20,6 +24,7 @@ def extract():
             headers={**COMMON_HEADERS},
             timeout=settings.REQUESTS_TIMEOUT,
         ).json()
+    log.warning("Missing required setting MICROMASTERS_CATALOG_API_URL")
```

Comment:
Some of the ETL tasks issue warnings for missing env vars already, others do not.

My understanding for why these are warnings not errors is that we want the celery tasks (which are run regularly in local dev and on rc/prod, and via management commands) to be no ops when the env vars are missing.